src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/ResolveElasticsearchStep.java

```diff
@@ -40,7 +40,15 @@ public void execute(InstanceConfiguration config)
                     .resolveArtifact(artifact.getArtifactCoordinates());
             unpackDirectory = getUnpackDirectory();
             ZipUtil.unpack(resolvedArtifact, unpackDirectory);
-            moveToElasticsearchDirectory(unpackDirectory, new File(config.getBaseDir()));
+            File baseDir = new File(config.getBaseDir());
+            moveToElasticsearchDirectory(unpackDirectory, baseDir);
+
+            String pathConf = config.getClusterConfiguration().getPathConf();
+            if ( pathConf != null && !pathConf.isEmpty() ) {
+                // Merge the user-defined config directory with the default one
+                // This allows user to omit some configuration files (jvm.options for instance)
+                FileUtils.copyDirectory( new File(pathConf), new File(baseDir, "config"));
+            }
```

Comment:
I had this on my mind for quite a while. They way I saw this being done was using the -Epath.conf argument to the ES instance (https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html).
Your solution should work better, for it keep the default config files if they haven't been defined in the provided config directory.